polus-feature-extraction-plugin/plugin.json

```diff
@@ -0,0 +1,160 @@
+		      { "description": "Median", "enum": [ "median" ] },
+                      { "description": "Mode", "enum": [ "mode" ] },
+		      { "description": "Standard Deviation", "enum": [ "standard_deviation" ] },
+		      { "description": "Skewness", "enum": [ "skewness" ] },
+                      { "description": "Entropy", "enum": [ "entropy" ] },
+		      { "description": "Kurtosis", "enum": [ "kurtosis" ] },
+                      { "description": "All", "enum": [ "all" ] }
+                  ], 
+              "default": "Area",
+              "widget": "select" ,    
+              "minItems": 1, 
+              "uniqueItems": "true"
+          },
+	"required": "true"
+	}
+      },
+      {
+            "name": "csvfile",
+            "type": "enum",
+            "options": {
+                "values": [
+                    "singlecsv",
+                    "separatecsv"
+                ]
+            },
+            "required": "true",
+            "description": "csvfile : singlecsv for saving values in one csv file and separate csv to save values for each image in separate csv file"
+      },
+      {
+	"name": "embeddedpixelsize",
+        "type": "boolean",
+        "description": "Consider embedded pixel sizes if present in metadata",
+        "required": "false"
+      },
+      {
+        "name": "unitLength",
+        "type": "string",
+        "description": "Enter the metric for unit conversion",
+        "required": "false"
+      },
+      {
+        "name": "pixelsPerunit",
+        "type": "number",
+        "description": "Enter the number of pixels per unit of the metric",
+        "required": "false"
+      }
+      
+    ],
+    "outputs": [
+      {
+        "name": "outDir",
+        "type": "collection",
+        "description": "Output collection"
+      }
+    ],
+    "ui": [
+      {
+        "key": "inputs.segDir",
+        "title": "Label image collection",
+        "description": "Labeled image collection"
+      },
+      {
+        "key": "inputs.intDir",
+        "title": "Intensity image collection",
+        "description": "Intensity image collection"
+      },
+      {
+        "key": "inputs.pixelDistance",
+        "title": "Pixel Distance",
+        "description": "Pixel distance to calculate the neighbors touching cells"
+      },
+      {
+        "key": "inputs.features",
+        "title": "Features",
+        "description": "Select features"
+      },
+      {
+        "key": "inputs.csvfile",
+        "title": "Output csv file",
+        "description": "Save csv file separately for each image or one csv for all images"
+      },      
+      {
+        "key": "inputs.embeddedpixelsize",
+        "title": "Embedded pixel size",
+        "description": "If selected, considers the embedded pixel size in the metadata"
+      },
+      {
+        "key": "inputs.unitLength",
+        "title": "Length of unit",
+        "description": "Enter the metric for unit conversion"
```

Comment:
The title is confusing. You do not want the user to specify the length of unit, but the actual units.